Makefile

```diff
@@ -73,18 +76,16 @@ deploy/charts/csi-driver/README.md: $(BINDIR)/helm-docs $(CHART_YAML)
 	$(BINDIR)/helm-docs
 
 .PHONY: depend
-depend: $(BINDIR)/helm $(BINDIR)/helm-docs
+depend: $(BINDIR)/helm $(BINDIR)/helm-docs $(BINDIR)/kind
 
 $(BINDIR)/helm: | $(BINDIR)
-	curl -o $(BINDIR)/helm.tar.gz -LO "https://get.helm.sh/helm-v$(HELM_VERSION)-$(OS)-$(ARCH).tar.gz"
-	tar -C $(BINDIR) -xzf $(BINDIR)/helm.tar.gz
-	cp $(BINDIR)/$(OS)-$(ARCH)/helm $(BINDIR)/helm
-	rm -r $(BINDIR)/$(OS)-$(ARCH) $(BINDIR)/helm.tar.gz
+		cd hack/tools && go build -o $(BINDIR)/helm helm.sh/helm/v3/cmd/helm
```

Comment:
I have the same concerns here as I mentioned in [this comment](https://github.com/cert-manager/approver-policy/pull/97#discussion_r934628551). I don't think there's a need to require that we download tool dependencies using `curl` but I think actually changing to `go build` is both unnecessary and honestly a step backwards.